odoo/addons/base/tests/test_translate.py

```diff
@@ -328,6 +342,57 @@ def test_105_duplicated_translation(self):
         self.assertEqual(len(translations), 1, "Translations were not duplicated after `translate_fields` call")
         self.assertEqual(translations.value, "Apprenti", "The first translation must stay")
 
+    def test_106_en_us_translation(self):
+        """ Test synchronizing translations with duplicated source """
+        # create a category with a French translation
+        cheese = self.env['res.partner.category'].create({'name': 'Cheese'})
+
+        self.env['ir.translation'].translate_fields('res.partner.category', cheese.id, 'name')
+
+        translations = self.env['ir.translation'].search([('name', '=', 'res.partner.category,name'), ('res_id', '=', cheese.id)], order='lang')
+        self.assertEqual(len(translations), 2)
+        self.assertRecordValues(translations,
+            [{'lang': 'en_US', 'src': 'Cheese', 'value': 'Cheese'},
+             {'lang': 'fr_FR', 'src': 'Cheese', 'value': 'Cheese'}])
+
+        # Translate in both language
+        translations[0].write({'value': 'The Cheese'})
+        translations[1].write({'value': 'Fromage'})
```

Comment:
Since these lines are updating one record each, maybe use setattr for clarity / simplicity?